spring-cloud-function-adapters/spring-cloud-function-adapter-gcp/pom.xml

```diff
@@ -17,14 +17,15 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<google.cloud.functions.version>1.0.0-alpha-2-rc3</google.cloud.functions.version>
+		<google.cloud.functions.api.version>1.0.0-alpha-2-rc3</google.cloud.functions.api.version>
```

Comment:
Should this be rc4?

Reply:
rc4 doesn't exist for the API.
See: https://github.com/GoogleCloudPlatform/functions-framework-java/issues/17